planet/cli/data.py

```diff
@@ -78,6 +81,18 @@ def check_item_types(ctx, param, item_types) -> Optional[List[dict]]:
         raise click.BadParameter(str(e))
 
 
+def check_item_type(ctx, param, item_type) -> Optional[List[dict]]:
+    '''Validates the item type provided by comparing it to all supported
+    item types.'''
+    if len(item_type.split(",")) > 1:
+        raise click.BadParameter("Only provide 1 item type.")
+    try:
+        validate_item_type(item_type)
+        return item_type
+    except SpecificationException as e:
+        raise click.BadParameter(str(e))
+
+
```

Comment:
Best practice is to move the return outside of the try/except block. This is for two reasons: 1. minimize the number of steps inside the `try` and 2. it's a little more readable when the return is at the top level and there is only one return.

```suggestion
    except SpecificationException as e:
        raise click.BadParameter(str(e))

    return item_type
```